### DISCUSSION TITLE
What classes does Tailwind use on the left Sidebar of their website?

### DISCUSSION THREAD

**Discussion by lamualfa** [tailwindlabs/tailwindcss#3499] [Help] [answered]
I didn't find any documentation for writing this class. is there any reference i can read?

![image](https://user-images.githubusercontent.com/39755201/106570152-18ae1c80-6568-11eb-869b-fdeb4d156b33.png)

`sticky?lg:h-(screen-18)`

What classes does Tailwind use on the left Sidebar of their website?

**Comment by adamwathan** [comment, marked as answer]
Hey! That's just a custom class we added to build that site 👍 not part of Tailwind.
